docs/references/javascript/organization-domain.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: OrganizationDomain
+description: The OrganizationDomain object is the model around an organization domain.
+---
+
+# `OrganizationDomain`
+
+The `OrganizationDomain` object is the model around an organization domain.
+
+## Attributes
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `id` | `string` | A unique identifier for this organization domain. |
+| `name` | `string` | The name for this organization domain (e.g example.com). |
+| `organizationId` | `string` | The organization ID of the organization this domain is for. |
+| `enrollmentMode` | `'manual_invitation' \| 'automatic_invitation' \| 'automatic_suggestion'` | An [enrollment mode][enrollment-mode-ref] will change how new users join an organization. |
+| `verification` | [`OrganizationDomainVerification`][org-domain-ver-ref] | The object that describes the status of the verification process of domain. |
+| `affiliationEmailAddress` | `string \| null` | The email address that was used to verify this organization domain. |
+| `totalPendingInvitations` | `number` | The number of total pending invitations sent to emails that match the domain name. |
+| `totalPendingSuggestions` | `number` | The number of total pending suggestions sent to emails that match the domain name. |
+| `createdAt` | `Date` | The date and time the membership was created. |
+| `updatedAt` | `Date` | The date and time the membership was updated. |
+
+### `OrganizationDomainVerification`
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `status` | `'unverified' \| 'verified'` | The status of the verification process. |
+| `strategy` | `'email_code'` | A string that indicates strategy of the verification. |
+| `attempts` | `number` | A number that indicates how many attempts have occured in order to verify the domain. |
+| `expiresAt` | `Date` | The expiration date and time of the verification. |
+
+
+## Methods
+
+### `delete()`
+
+Deletes the organization domain and removes it from the organization.
+
+```typescript
+function delete: () => Promise<void>;
+```
+
+
+## `prepareAffiliationVerification()`
+
+```typescript
+function prepareAffiliationVerification(params: PrepareAffiliationVerificationParams): Promise<OrganizationDomain>;
+```
+
+Begins the verification process of a created organization domain. This is a required step in order to complete the registration of the domain under the organization.
+
+### `PrepareAffiliationVerificationParams`
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `affiliationEmailAddress` | `string` | An email address that is affiliated with the domain name (e.g. user@example.com). |
+
+### Returns
+
+| Type | Description |
+| --- | --- |
+| <code>Promise\<[OrganizationDomain][org-domain-ref]\></code> | This method returns a <code>Promise</code> which resolves with a <code>[OrganizationDomain][org-domain-ref]</code> object. |

```

Comment:
if the user is already on this page, no need to link to a reference - they are already on the reference page.

```suggestion
| `Promise<OrganizationDomain>` | This method returns a `Promise` which resolves with an `OrganizationDomain` object. |

```